### PR TITLE
Need Test

### PR DESCRIPTION
**Pull request by Vincent-Michael**

### CODE REVIEW

File: WowPacketParser/Misc/Settings.cs [hunk 1]
```diff
@@ -45,33 +45,17 @@ public static class Settings
         public static readonly string HotfixesDatabase = Conf.GetString("HotfixesDatabase", "hotfixes");
         public static readonly string CharacterSet = Conf.GetString("CharacterSet", "utf8");
 
-        private static UInt64 GetSQLOutputFlag()
+        private static List<string> GetSQLOutput()
```
Comment (DDuarte) [inline WowPacketParser/Misc/Settings.cs]: Why is this a `List<string>` instead of `List<SQLOutput>`? All those `ToString()` are unnecessary.

Comment (Vincent-Michael) [inline WowPacketParser/Misc/Settings.cs]: oh, y xD

**Comment by DDuarte** [comment]
Why is the PR and commit message named "Need Test"?

What is wrong with the existing code?

**Comment by Vincent-Michael** [comment]
some enable stuff has no sqlout (with valid data)

**Comment by DDuarte** [comment]
Do not merge this until `sql_rewrite` is added to master.

**Comment by msoky** [comment]
what is this?

**Pull request closed without merging**